app/services/data_migrations/remove_unconditional_offers_via_api_feature_flag.rb

```diff
@@ -0,0 +1,10 @@
+module DataMigrations
```

Comment:
Looks good Avin! Could we split this into two PRs please – the migration should follow in a subsequent migration 🙏

Reply:
Sure, no problem. 

What's the reason for this?

Reply:
Ok fair point.

Moved the data migration to this PR: https://github.com/DFE-Digital/apply-for-teacher-training/pull/8987, will merge after this PR is merged and a rebase has been done.